parsers/whichbrowser-php-2/scripts/parse.php

```diff
@@ -58,7 +58,7 @@
                 'name'     => !empty($r->device->model) ? $r->device->model : '',
                 'brand'    => !empty($r->device->manufacturer) ? $r->device->manufacturer : '',
                 'type'     => !empty($r->device->type) ? $r->device->type : '',
-                'ismobile' => $r->isMobile() ? true : false,
+                'ismobile' => $r->isMobile() ? 'true' : 'false',
```

Comment:
Do all parsers return these values as strings?

Reply:
Hmm, no, but in this case I was seeing differences between this parser and the whichbrowser test suite because the test suite was expecting a string.

I'll look over all parsers and tests and make sure they're all using one or the other (whichever the majority is at this point).